install/update_92_93.php

```diff
@@ -47,6 +47,37 @@ function update92to93() {
    $migration->displayTitle(sprintf(__('Update to %s'), '9.3'));
    $migration->setVersion('9.3');
 
+   //Create solutions table
+   if (!$DB->tableExists('glpi_inventorytypes')) {
+      $query = "CREATE TABLE `glpi_inventorytypes` (
+         `id` int(11) NOT NULL AUTO_INCREMENT,
+         `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+         `comment` text COLLATE utf8_unicode_ci,
+         `date_creation` datetime DEFAULT NULL,
+         `date_mod` datetime DEFAULT NULL,
+         PRIMARY KEY (`id`),
+         KEY `name` (`name`)
+         ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+      $DB->queryOrDie($query, "9.3 add table glpi_inventorytypes");
+   }
+
+   //Add automatic inventory infos for each itemtype
+   $fields = [
+      'date_last_inventory' => 'datetime', //Last inventory
+      'date_last_seen'      => 'datetime', //Last time the asset has been seen (either local or remote inventory)
+      'inventorytypes_id'   => 'integer', //Type of inventory (fusion, ocs, etc)
+      'inventory_source'    => 'string' //Which device has discovered the asset
+   ];
+   foreach ($CFG_GLPI["asset_types"] as $itemtype) {
```

Comment:
Since asset_type may evolve in the future, it would be better to list current asset types ;)